docs/installation/server-config/index.rst

```diff
@@ -1,172 +1,233 @@
 .. _ksql-server-config:
 
 Configuring KSQL Server
------------------------
+=======================
 
-.. toctree::
+.. toctree:: Contents
 
     security
     ccloud-ksql
     avro-schema
 
-You can set the default properties for KSQL, :cp-javadoc:`Kafka’s Streams |streams/javadocs/index.html`, Kafka’s
-:cp-javadoc:`producer client |clients/javadocs/org/apache/kafka/clients/producer/ProducerConfig.html` and
-:cp-javadoc:`consumer client |clients/javadocs/org/apache/kafka/clients/consumer/ConsumerConfig.html`, and admin client properties in the ``/etc/ksql/ksql-server.properties`` file.
+KSQL configuration parameters can be set for Admin Client, KSQL Query, KSQL Server, Kafka Streams, and Kafka Client. The
+KSQL Server parameters are configured via the ``/etc/ksql/ksql-server.properties`` file. The KSQL CLI parameters are
+configured via the ``/etc/ksql/ksql-server.properties`` file or the SET statement in the KSQL CLI.
```

Comment:
I think only the streams properties can be set with the `SET` command, not the server parameters.. the latter are set at startup and can only be in the properties file.

Reply:
so the KSQL CLI can only modify Streams parms - and these can only be modified using the SET property...so `ksql-server.properties` is only applicable to KSQL server?

Reply:
`ksql-server.properties` includes all properties: ksql server, streams, kafka clients, ksql engine,.

The properties you can set in the CLI are _only_ the streams. properties. If you pass the same `ksql-server.properties` file to the server and the CLI, the CLI will only effectively use the streams properties from that file.